backend/eventd/integration_test.go

```diff
@@ -75,6 +76,12 @@ func TestEventdMonitor(t *testing.T) {
 	event.Check.Interval = 1
 	event.Check.Ttl = 5
 
+	ctx := otherTestutil.ContextWithNamespace("default")(context.Background())
```

Comment:
How many ways do we have to create a context with a namespace key? :(
https://github.com/sensu/sensu-go/blob/master/backend/store/namespace.go#L25-L28
https://github.com/sensu/sensu-go/blob/master/testing/testutil/context.go#L28-L34

Reply:
About as many `package testutil`s I think